cmd/client/main.go

```diff
@@ -87,6 +82,11 @@ func main() {
 			Value: "/var/run/ocid.sock",
 			Usage: "Socket to connect to",
 		},
+		cli.DurationFlag{
+			Name:  "timeout",
+			Value: 10 * time.Second,
```

Comment:
Maybe we should make the timeout in units of seconds. Although, I'm not sure how `DurationFlag` works so maybe this isn't an issue?